portal/sign/assets/signer_modal.tpl.php

```diff
@@ -8,37 +8,59 @@
  * @copyright Copyright (c) 2016-2019 Jerry Padgett <sjpadgett@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
-require_once dirname(__FILE__) . "/../../../vendor/autoload.php";
-require_once(dirname(__FILE__) . '/../../../library/sql.inc');
 
-$isAdmin = !empty($_GET['isAdmin']) ? 'checked' : '';
+require_once(dirname(__FILE__) . "/../../../src/Common/Session/SessionUtil.php");
+OpenEMR\Common\Session\SessionUtil::portalSessionStart();
+
+$is_portal = isset($_SESSION['portal_init']) ? true : $_GET['isPortal'];
+if (!$is_portal) {
+    session_destroy();
+    require_once(dirname(__FILE__) . '/../../../interface/globals.php');
+} else {
+    require_once dirname(__FILE__) . "/../../verify_session.php";
+}
+
+$cuser = js_escape(isset($_SESSION['authUser']) ? $_SESSION['authUser'] : "-patient-");
+$cpid = js_escape(isset($_SESSION['pid']) ? $_SESSION['pid'] : "0");
```

Comment:
Above 2 lines look odd since are escaping the conditional.

Reply:
Disregard comment; I missed the parenthesis. Looks good.

Reply:
Looks like above 2 variables are being used in html context and not js context. So, would either use attr() instead above or preferable, just escape it with attr when use it below in the html output.